src/Extension/Symfony.php

```diff
@@ -58,4 +58,25 @@ public static function assertJsonValueEquals($expected, $expression, $response)
     {
         Assert::assertJsonValueEquals($expected, $expression, json_decode($response->getContent()));
     }
+
+    /**
+     * Asserts that a response is successful and of type json.
+     *
+     * @param Response $response   Response object
+     * @param int      $statusCode Expected status code (default 200)
+     *
+     * @see \Bazinga\Bundle\RestExtraBundle\Test\WebTestCase::assertJsonResponse()
+     */
+    public static function assertJsonResponse(Response $response, $statusCode = 200)
+    {
+        \PHPUnit_Framework_Assert::assertEquals(
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\PHPUnit_Framework_Assert' in method 'assertJsonResponse'.](https://www.codacy.com/app/estahn/phpunit-json-assertions/file/2281688906/issues/source?bid=3181605&fileBranchId=3188493#l72)